chart/validator/values.yaml

```diff
@@ -275,6 +351,26 @@ pluginSecrets:
     # secretAccessKey: ""
     # sessionToken: ""
 
+  # If installing the Azure plugin, the below config is required unless one of the following applies:
+  # - The target cluster already contains a secret with the correct format and you've specified its name above.
+  #   If specifying your own secret, refer to chart/validator/templates/plugin-secret-azure.yaml for
+  #   an example of the expected format and ensure that it exists in the same namespace as the Helm release.
+  # - You're deploying to a K8s cluster in AKS and relying on WorkloadIdentityCredentials, plus you've specified auth.serviceAccountName
+  #
+  azure: {}  # Don't forget to delete these curly braces if you're specifying credentials here!
+    # secretName: azure-creds  # ensure this matches the Azure plugin values above
+    #
+    # Any valid combination of Azure credential environment variables, per the options defined in:
+    # https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#readme-environment-variables.
+    # env:
+    #   AZURE_CLIENT_ID: ""
+    #   AZURE_TENANT_ID: ""
+    #   AZURE_CLIENT_SECRET: ""
+    #   AZURE_CLIENT_CERTIFICATE_PATH: ""
+    #   AZURE_CLIENT_CERTIFICATE_PASSWORD: ""
+    #   AZURE_USERNAME: ""
+    #   AZURE_PASSWORD: ""
+
   # If installing the vSphere plugin, the below config is required unless the following applies:
```

Comment:
looks like some refs to vsphere left over

Reply:
(otherwise, whole PR lgtm)

Reply:
@mattwelke the vSphere content is meant to be there.